ci-operator/config/windup/windup-ui-tests/windup-windup-ui-tests-main__mtr-ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,52 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: dockerfiles/interop/Dockerfile
+  to: mtr-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: mtr-interop-aws
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: aws.interop.ccitredhat.com
+      OPERATORS: |
```

Comment:
Yeah that can be done, either way works. I can update it today

Reply:
@ascerra So, we can do this, but `make update` will update it to two lines, probably because it is surpassing a line-length rule.

Here is what it would look like:

```yaml
      OPERATORS: '[{"name": "mtr-operator", "source": "redhat-operators", "channel":
        "alpha", "install_namespace": "mtr", "target_namespaces": "mtr"}]'
```

I think I like seeing the operator definition on one line, the way it currently is. Any thoughts?

Reply:
Ya if we can't do this on a single line then I think what you've already done here works best. Thanks for looking into this!